GiniVision/Classes/Core/GiniVisionUtils.swift

```diff
@@ -70,7 +70,9 @@ internal func UIFontPreferred(_ weight: FontWeight, andSize size: CGFloat) -> UI
 
 internal struct AnimationDuration {
     static var slow = 1.0
+    static var mediumSlow = 0.7
```

Comment:
Do we really need this granularity? Three speeds should be enough, I mean when do I use mediumSlow? Or should I use slow? Maybe mediumFast is better... no it's too slow I should use fast instead... Can be confusing to other devs.

Reply:
Honestly when I wrote this I thought it was a good idea, thinking about different animation speeds. But for regular animations doesn't make sense. I think I will revert it to the previous values, with the recommended value Google guidelines.